src/Server/Grant/PasswordGrant.php

```diff
@@ -110,8 +110,7 @@ public function createTokenResponse(
             throw OAuth2Exception::invalidRequest('Username and/or password is missing');
         }
 
-        $callback = $this->callback;
-        $owner    = $callback($username, $password);
+        $owner = call_user_func_array($this->callback, [$username, $password]);
```

Comment:
Why this change? Directly calling the method is much more performant than call_user_func_array :).